src/schema/resolvers.js

```diff
@@ -193,11 +193,12 @@ function buildTransactionFilters({
  * @param {oracle} oracle
  */
 const getPhase = ({ token, status }) => {
-  if (token === 'QTUM' && status === 'VOTING') return 'betting';
+  if (token === 'QTUM' && ['VOTING', 'PENDING', 'CREATED'].includes(status)) return 'betting';
```

Comment:
sorry I believe I gave you bad info. `QTUM` + `CREATED` & `VOTING` statuses are for the betting phase.
`PENDING` only happens when an Oracle's result gets set and its waiting for the finalize result call where all the Oracles for that event will go into `WITHDRAW`

Reply:
so the logic should be 
```js
const getPhase = ({ token, status }) => {
  const [BOT, QTUM] = [token === 'BOT', token === 'QTUM'];
  if (QTUM && ['VOTING', 'CREATED'].includes(status)) return 'betting';
  if (BOT && status === 'VOTING') return 'voting';
  if (QTUM && ['WAITRESULT', 'OPENRESULTSET'].includes(status)) return 'resultSetting';
  if (BOT && status === 'WAITRESULT') return 'finalizing';
  if ((BOT || QTUM) && status === 'WITHDRAW' || QTUM && status === 'PENDING') return 'withdrawing';
  throw Error(`Invalid Phase determined by these -> TOKEN: ${token} STATUS: ${status}`)
};
```
???

Reply:
https://github.com/bodhiproject/bodhi-server/pull/24